nemo/collections/common/losses/aggregator.py

```diff
@@ -17,14 +17,14 @@
 import torch
 from torch import nn
 
-from nemo.core.classes import NeuralModule, Typing, typecheck
+from nemo.core.classes import Serialization, Typing, typecheck
 from nemo.core.neural_types import LabelsType, LengthsType, LogitsType, LogprobsType, LossType, MaskType, NeuralType
 from nemo.utils.decorators import experimental
 
 __all__ = ['AggregatorLoss']
 
 
-class AggregatorLoss(NeuralModule):
+class AggregatorLoss(nn.Module, Typing, Serialization):
     """
     Neural module which combines sums several losses into one.
     Args:
```

Comment:
'Neural module' - can we still call a neural module? 
`combines sums`

Reply:
i would not , its still a loss, it takes nemo.core.loss are inputs. so it should be that too. modules need restore and save_to which doesnt make sense here

Reply:
exactly, my comment was regarding description please delete 'neural modules' and either 'combines' or 'sums'